sunpy/net/dataretriever/sources/sept.py

```diff
@@ -0,0 +1,74 @@
+from sunpy.util.scraper import Scraper
+from sunpy.time import TimeRange
+import datetime
+
+def _get_url_for_timerange(timerange = TimeRange('2007-01-20','2015-01-01'), stereo_spacecraft = 'ahead', 
+						   duration_of_average = '10min', specie = 'ele', sensor_pointing = 'asun'):
+	"""
+	Returns list of URLS to STEREO SEPT data files corresponding to value of input timerange.
+	URL Source : http://www2.physik.uni-kiel.de/stereo/data/sept/level2/
+
+	The earliest data available is from 20-Jan-2007.
+
+	Parameters
+	----------
+	timerange: sunpy.time.TimeRange
+	    time range for which data is to be downloaded.
+	    Default value -  TimeRange('2007-01-20','2015-01-01')	
```

Comment:
I don't believe a timerange by default is a good idea. I'd rather show an example in the documentation.